app/shared/services/contentful-content.service.ts

```diff
@@ -66,11 +77,11 @@ export class ContenfulContent {
    * @param slug
    * @returns {any}
    */
-  getNodePage(slug: string): Observable<NodePageContent> {
+  getNodePage(slug: string): Observable < NodePageContent > {
```

Comment:
In some places you have spaces between angle brackets in some - don't like in the code below
`transformResponse<ContentfulNodePage>`